Packs/Microsoft365Defender/ReleaseNotes/3_1_0.md

```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### Microsoft 365 Defender
+- Add command: `!microsoft-365-defender-incident-get`
+- Updated the Docker image to: *demisto/crypto:1.0.0.27590*.
```

Comment:
Please change the value here to the right updated docker image tag.

Reply:
Where can I find correct value?

Reply:
In the yml file - it is `demisto/crypto:1.0.0.28507`

Reply:
done